backend/app/api/v1/endpoints/auth.py

```
@@ -61,12 +61,12 @@ async def login_endpoint(
     summary="Refresh access token",
     description="Generate new access token using a valid refresh token."
 )
-@require_permissions([Permission.REFRESH_TOKEN])
 async def refresh_token_endpoint(
     request: Request,
     token_request: RefreshTokenRequest,
     db: AsyncSession = Depends(get_db),
-    settings: Settings = Depends(get_settings)
+    settings: Settings = Depends(get_settings),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)